notebooks/py_scripts/10_Spike_SortingV1.py

```diff
@@ -94,13 +101,20 @@
 
 sgs.ArtifactDetection()
 
-# insert SpikeSortingSelection. again use `insert_selection` method
+# insert SpikeSortingSelection. again use `insert_selection` method.
+#
+# We tested mountainsort4, mountainsort5, kilosort2_5, kilosort3, and ironclust.
+# when using mountainsort5, pip install 'mountainsort5'
+# when using Kilosorts and ironclust -- make sure to pip install 'cuda-python' and 'spython'
+# For sorting with Kilosort, make sure to use a machine with GPU and put the whole probe not a sliced individual shank.
 #
 
-key = {
+# +
+sorter = "kilosort2_5"
```

Comment:
@donghoon-shin is this using kilosort2.5 in the tutorial? I think we should use mountainsort4.